docs/MAVLINK_FOR_OPENTX.md

```diff
@@ -6,58 +6,62 @@ This page describes how to set up a mLRS system for OpenTx radios running the MA
 
 Four steps need to be completed:
 1. The OpenTx radio needs to be flashed with the MAVLink for OpenTx firmware.
-2. The mLRS Tx module needs to be put into "mBridge mode".
-3. The flight controller needs to be set up for MAVLink on a serial port.
-4. The radio needs to be configured. 
+2. The radio needs to be configured. 
+3. The mLRS Tx module needs to be put into "mBridge mode".
+4. The flight controller needs to be set up for MAVLink on a serial port.
 
 Step 1. is beyond the scope of this article; please consult the project's discussion channels.
 
-In principle, there is no specific configuration of the mLRS receiver neccessary. It is however recommended to set the receiver into "mavlink mode" and to use the CRSF protocol, as described below.
+In principle, there is no specific configuration of the mLRS receiver neccessary. It is however recommended to set the receiver into "MAVLink mode" and to use the CRSF protocol, as described below.
```

Comment:
This seems to contradict step 4 above which says the FC needs to be setup for MAVLink.  I would guess that the purpose of using MAVLink for OpenTx _is_ to have MAVlink on the FC and use MAVLink mode, so I wonder if this line can be removed.

Reply:
haha ... you do have a point here :D

Reply:
done